cli/runtime/docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Tanzu CLI Integration Library
+
+This library exposes many different functionalities that plugin developers can use while developing Tanzu CLI plugins. Below are some of the functionalities that are available:
```

Comment:
suggest:
This library provides functionality useful in the development of Tanzu CLI plugins. It includes: